Week-1/aryan29/README.md

```diff
@@ -0,0 +1,60 @@
+## Problem 1 - Ankit's Phone
+
+This is a python program which can get you JSON of features of any number of mobiles in simple steps and store it in file named "data.json" and provide you with regular updates about decrease in price of mobiles mentioned
+
+## Usage
+
+The program initially comes with a "user_config.json" file in which you can add your custom settings to get started with using the product and make sure to add smartprix code of the product you want to keep track of after that just type this to get your json file
+
+```python
+python api.py -j
+```
+You can add mysmartprice code too as a second code for keeping track of best price from both of the websites
+![Example UserConfig.json](https://github.com/aryan29/Git-Freeze/blob/aryan29/week1/Week-1/aryan29/images/user_config.png)
```

Comment:
Not coding related but using relative paths for images would be better. 

Since I cloned the repo and locally (w/o internet connectivity) I can't render .md right?